modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -73,22 +73,40 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					<aui:button cssClass="btn btn-primary btn-sm my-2" href="<%= cpContentHelper.getReplacementCommerceProductFriendlyURL(cpSku, themeDisplay) %>" value="replacement-product" />
 				</c:if>
 
-				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
-
-					<%
-					Format format = FastDateFormatFactoryUtil.getSimpleDateFormat("MMMMM dd, yyyy", locale, timeZone);
-					%>
+				<%
+				Format dateFormatDateTime = FastDateFormatFactoryUtil.getDateTime(locale, timeZone);
+				%>
 
+				<c:if test="<%= (cpSku != null) && (cpSku.getDiscontinuedDate() != null) %>">
 					<p class="my-2">
 						<span class="font-weight-semi-bold">
 							<%= LanguageUtil.get(request, "end-of-life") %>
 						</span>
 						<span>
-							<%= format.format(cpSku.getDiscontinuedDate()) %>
+							<%= dateFormatDateTime.format(cpSku.getDiscontinuedDate()) %>
 						</span>
 					</p>
 				</c:if>
 
+				<c:choose>
+					<c:when test="<%= cpSku != null %>">
+						<% CommerceInventoryReplenishmentItem commerceInventoryReplenishmentItems =
```

Comment:
It should be singular as it is a single object

Reply:
The variable is `commerceInventoryReplenishmentItems` but it's a single object, so it should be `commerceInventoryReplenishmentItem`